src/commands/copy/copy.ts

```diff
@@ -71,8 +85,8 @@ export const commandCopy = command({
             manifest,
             start: i,
             size: chunkSize,
-            force: args.force,
-            noClobber: args.noClobber,
+            force: force,
```

Comment:
you don't need the `: force` if the key and value are the same name.

I should add a lint rule to auto fix this.